src/panels/lovelace/editor/config-elements/hui-glance-card-editor.ts

```diff
@@ -89,6 +90,10 @@ export class HuiGlanceCardEditor extends LitElement
     return this._config!.show_state || true;
   }
 
+  get _state_color(): boolean {
+    return Boolean(this._config!.state_color);
+  }
```

Comment:
For glance state color is default true right? So should be:
```suggestion
  get _state_color(): boolean {
    return this._config!.state_color ?? true;
  }
```